public/Day 178/script.js

```
@@ -0,0 +1,970 @@
+    }
+}
+
+// Handle mouse up
+function handleMouseUp() {
+    mouseDown = false;
+}
+
+// Set current tool and update UI
+function setTool(tool) {
+    currentTool = tool;
+    document.querySelectorAll('.tool-btn').forEach(btn => btn.classList.remove('active'));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)